src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -1548,10 +1548,40 @@ public void Copy()
         }
 
         /// <summary>
-        ///     Paste ISelectable objects from the clipboard to the workspace.
+        ///     Paste ISelectable objects from the clipboard to the workspace 
+        /// so that the nodes appear in their original location with a slight offset
         /// </summary>
         public void Paste()
         {
+            // Provide a small offset when pasting so duplicate pastes aren't directly on top of each other
+            CurrentWorkspace.IncrementPasteOffset();
+
+            var locatebleModels = ClipBoard.Where(model => model is NoteModel || model is NodeModel);
```

Comment:
Won't this get rid of annotation models? (i.e. groups) is there a reason we can't check that the model is not ILocatable instead?

is there a problem with measuring groups in this way to calculate the bounds?